code-challenges.test.js

```diff
@@ -14,46 +14,224 @@
+//     Test Suites: 1 passed, 1 total
+//     Tests:       1 passed, 1 total
+
+
 
 // b) Create the function that makes the test pass.
 
+// A function that takes in an array;
+    // It returns an array with all the numbers multiplied by 3
+const multiplyBy3 = (array) => {
+    return array.map(value => value*3)}
+
+
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+/*
+    A describe block that describes a function that
+        takes an object as an argument
+        decides if the number inside it is evenly divisible by 3 or not
+
+    Errors encountered:
+
+        ReferenceError: divisibleBy3Evenl is not defined
+        SyntaxError: Missing initializer in const declaration.
+        TypeError: divisibleBy3Evenly(...).toEqual is not a function
+*/
+
+describe("divisibleBy3Evenly" , () => {
+    // The 'it' block 
+    it ("takes a object as an argument and decides if the number inside it is evenly divisible by three or not" , () => {
+        
+        // Use provided test inputs
+        const object1 = { number: 15 }
+        // Expected output: "15 is divisible by three"
+        const output1 = "15 is divisible by three"
+
+        const object2 = { number: 0 }
+        // Expected output: "0 is divisible by three"
+        const output3 = "0 is divisible by three"
+
+        const object3 = { number: -7 }
+        // Expected output: "-7 is not divisible by three"
+        const output4 = "-7 is not divisible by three"
+
+        // Expect statements using provided inputs/outputs
+        expect (divisibleBy3Evenly(object1)).toEqual(output1)
+
+    })
+})
 
 // b) Create the function that makes the test pass.
+/* 
+    Write a function that;
+        accepts an object as an argument
+            if the number in the object is evenly divisible by 3
+                return string stating so
+            if the number in the object is not evenly divisible by 3
+                return string stating so
+
+
+    Errors Encountered:
+    Received: "undefined is not evenly divisible by three"
+    Test Suites: 1 failed, 1 total
+    Tests:       1 failed, 1 passed, 2 total
+
+
+    Final output:
+    Test Suites: 1 passed, 1 total
+    Tests:       1 passed, 1 total
+*/
+
+// A function that takes in an object
+    // Checks if the number property is evenly divisible by 3
+    // Returns a string
+const divisibleBy3Evenly = (object1) => {
+
+    // Use an if-else conditional statement
+    // To check for % 3 === 0
+    if (object1.number % 3 === 0){
+        let x = object1.number.toString()
```

Comment:
I don't see you using x inside this function so it may not be needed in here.  One thing to consider is currently you have hardcoded that if a number is divisible by three it will output `15 is divisible by three`, but what if the number is another number that is divisible by three?